### DISCUSSION TITLE
Mono theme: Predefine color schemes

### DISCUSSION THREAD

**Discussion by Monkeyboy307** [GetPublii/Publii#1605] [Help] [answered]
I've downloaded and installed the Mono theme from here: [https://marketplace.getpublii.com/themes/mono/](https://marketplace.getpublii.com/themes/mono/)

On the page I've linked, it says the following...

> **Color Flexibility:** Choose from seven predefined color schemes or create your own custom color palette to match your branding or personal style preferences.

However, I can't seem to find where these options are. The only option seems to be to change a single accent color under the colors section of the theme's settings.

Is there such an option for seven predefined color schemes? Or is it mistake on the theme's download page?

**Comment by bobmitro** [comment, marked as answer]
There may be a discrepancy in the theme's download page description. The current functionality of the Mono theme indeed allows for the customization of a single accent color under the colors section of the theme's settings.
